src/vs/workbench/contrib/customEditor/browser/commands.ts

```diff
@@ -172,3 +172,54 @@ MenuRegistry.appendMenuItem(MenuId.CommandPalette, {
 		model.redo();
 	}
 }).register();
+
+(new class ToggleCustomEditorCommand extends Command {
+	public static readonly ID = 'editor.action.customEditor.toggle';
+
+	constructor() {
+		super({
+			id: ToggleCustomEditorCommand.ID,
+			precondition: CONTEXT_HAS_CUSTOM_EDITORS,
+			kbOpts: {
+				primary: KeyMod.CtrlCmd | KeyCode.KEY_E,
+				weight: KeybindingWeight.EditorContrib
+			}
+		});
+	}
+
+	public runCommand(accessor: ServicesAccessor): void {
+		const editorService = accessor.get<IEditorService>(IEditorService);
+		const activeControl = editorService.activeControl;
+		if (!activeControl) {
+			return;
+		}
+
+		const activeGroup = activeControl.group;
+		const activeEditor = activeControl.input;
+		if (!activeEditor) {
+			return;
+		}
+
+		const targetResource = activeEditor.getResource();
+		if (!targetResource) {
+			return;
+		}
+
+		const customEditorService = accessor.get<ICustomEditorService>(ICustomEditorService);
+		const viewIDs = customEditorService.getContributedCustomEditors(targetResource);
+
+		const activeCustomEditor = customEditorService.activeCustomEditor;
+		let toggleView = 'default';
```

Comment:
[nit] Get the default view id from the `defaultEditorInfo` (you may need to export that constant too)